app_dart/test/service/luci_build_service_test.dart

```diff
@@ -303,7 +303,6 @@ void main() {
       config = FakeConfig(deviceLabServiceAccountValue: serviceAccountInfo);
       mockBuildBucketClient = MockBuildBucketClient();
       service = LuciBuildService(config, mockBuildBucketClient, serviceAccountInfo);
-      config.luciTryInfraFailureRetriesValue = 3;
```

Comment:
Same here why are we removing the retries config?

Reply:
Ditto.